packages/web/src/app/doc-pages/brand/illustrations-doc/illustrations-exhibit/illustrations-exhibit.component.ts

```diff
@@ -0,0 +1,90 @@
+import { animate, style, transition, trigger } from '@angular/animations';
+import { Overlay, OverlayModule } from '@angular/cdk/overlay';
+import { CdkPortal, PortalModule } from '@angular/cdk/portal';
+import { AsyncPipe } from '@angular/common';
+import { Component, HostListener, ViewChild } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+
+import { IllustrationsExhibitDetailsComponent } from './illustrations-exhibit-details/illustrations-exhibit-details.component';
+import { IllustrationsExhibitFilterComponent } from './illustrations-exhibit-filter/illustrations-exhibit-filter.component';
+import { IllustrationsExhibitListComponent } from './illustrations-exhibit-list/illustrations-exhibit-list.component';
+import { IllustrationsExhibitService } from './illustrations-exhibit.service';
+import { BreakpointService } from 'src/app/core/services/breakpoint.service';
+
+@Component({
+  selector: 'app-illustrations-exhibit',
+  standalone: true,
+  imports: [
+    AsyncPipe,
+    OverlayModule,
+    PortalModule,
+    IllustrationsExhibitFilterComponent,
+    IllustrationsExhibitListComponent,
+    IllustrationsExhibitDetailsComponent,
+  ],
+  templateUrl: './illustrations-exhibit.component.html',
+  animations: [
+    trigger('entranceAnimation', [
+      transition(':enter', [
+        style({ opacity: 0, translate: '0 24px' }),
+        animate('200ms ease-in', style({ opacity: 1, translate: '0' })),
+      ]),
+      transition(':leave', [animate('200ms ease-in', style({ opacity: 0, translate: '0 24px' }))]),
+    ]),
+  ],
+})
+export class IllustrationsExhibitComponent {
+  @ViewChild(CdkPortal) portal: CdkPortal;
+
+  constructor(
+    private illustrationsExhibitService: IllustrationsExhibitService,
+    overlay: Overlay,
+    breakpointService: BreakpointService,
+  ) {
+    const overlayRef = overlay.create({
+      positionStrategy: overlay.position().global().centerHorizontally().bottom('16px'),
+      maxWidth: 'calc(100vw - 32px)',
+      scrollStrategy: overlay.scrollStrategies.reposition(),
+    });
```

Comment:
Despite that I love the CDKPortal for creating connected overlays, I think we could implement this without the CDK Portal, since the position is fixed bottom 😄

Reply:
Fair point, looking at it now it's definitely not needed! I'll refactor